docs/object.md

```diff
@@ -162,6 +167,7 @@ getRotationValues() | Returns a Table of rotation values. Rotation values are us
 <a class="anchor" id="getstateid"></a>getStateId() | Current [state](https://kb.tabletopsimulator.com/host-guides/creating-states/) ID (index) an object is in. Returns -1 if there are no other states. State ids (indexes) start at 1. | [<span class="ret int"></span>](types.md) |
 getStates() | Returns a Table of information on the [states](https://kb.tabletopsimulator.com/host-guides/creating-states/) of an Object. | [<span class="ret tab"></span>](types.md) | [<span class="i"></span>](#getstates)
 getValue() | Object value. What the value represents depends on what type of Object this function is used on. | [<span class="ret int"></span>](types.md) | [<span class="i"></span>](#getvalue)
+<a class="anchor" id="isdestroyed"></a>isDestroyed() | Returns true if an object is (or will be) destroyed. | [<span class="ret boo"></span>](types.md) |
```

Comment:
Object, capital O

Reply:
Should all objects on this page be updated to Object because right now they are inconsistent?